src/main/java/no/nav/pto/veilarbportefolje/domene/Bruker.java

```diff
@@ -58,6 +60,7 @@
     LocalDateTime nesteAktivitetStart;
     LocalDateTime forrigeAktivitetStart;
     LocalDateTime oppfolgingStartdato;
+    LocalDateTime nesteUtlopsdatoAktivitet;
```

Comment:
Since this class is growing, maybe we can consider to move some things during future refactoring in seperate class (e.g. related to siste endring), and include that class as child here...